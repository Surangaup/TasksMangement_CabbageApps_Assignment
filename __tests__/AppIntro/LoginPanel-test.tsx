import * as React from 'react';
import renderer from 'react-test-renderer';
import LoginPanel from '../../src/screens/app_intro/loginPanel';

test(('LoginPanel snapshot'), ()=>{
    const snap = renderer.create(<LoginPanel />).toJSON();
    expect(snap).toMatchSnapshot();
})