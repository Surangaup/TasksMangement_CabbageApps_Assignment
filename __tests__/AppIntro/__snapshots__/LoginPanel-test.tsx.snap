// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginPanel snapshot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "white",
    }
  }
>
  <RNGestureHandlerButton
    collapsable={false}
    onGestureEvent={[Function]}
    onGestureHandlerEvent={[Function]}
    onGestureHandlerStateChange={[Function]}
    onHandlerStateChange={[Function]}
    rippleColor={0}
  >
    <View
      accessible={true}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#f9f9fb",
          "borderRadius": 15,
          "flexDirection": "row",
          "height": 50,
          "marginHorizontal": "5%",
          "marginVertical": 5,
          "opacity": 1,
          "padding": 10,
        }
      }
    >
      <Image
        height={20}
        source={
          Object {
            "testUri": "../../../src/asserts/images/facebook.png",
          }
        }
        style={
          Object {
            "marginLeft": 5,
          }
        }
        width={20}
      />
      <Text
        style={
          Object {
            "flex": 1,
            "fontFamily": "Gilroy SemiBold",
            "fontSize": 15,
            "paddingLeft": 5,
            "textAlign": "center",
          }
        }
      >
        Continue with Facebook
      </Text>
    </View>
  </RNGestureHandlerButton>
  <RNGestureHandlerButton
    collapsable={false}
    onGestureEvent={[Function]}
    onGestureHandlerEvent={[Function]}
    onGestureHandlerStateChange={[Function]}
    onHandlerStateChange={[Function]}
    rippleColor={0}
  >
    <View
      accessible={true}
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#f9f9fb",
          "borderRadius": 15,
          "flexDirection": "row",
          "height": 50,
          "marginHorizontal": "5%",
          "marginVertical": 5,
          "opacity": 1,
          "padding": 10,
        }
      }
    >
      <Image
        height={20}
        source={
          Object {
            "testUri": "../../../src/asserts/images/google.png",
          }
        }
        style={
          Object {
            "marginLeft": 5,
          }
        }
        width={20}
      />
      <Text
        style={
          Object {
            "flex": 1,
            "fontFamily": "Gilroy SemiBold",
            "fontSize": 15,
            "paddingLeft": 5,
            "textAlign": "center",
          }
        }
      >
        Continue with Google
      </Text>
    </View>
  </RNGestureHandlerButton>
  <RNGestureHandlerButton
    collapsable={false}
    onGestureEvent={[Function]}
    onGestureHandlerEvent={[Function]}
    onGestureHandlerStateChange={[Function]}
    onHandlerStateChange={[Function]}
    rippleColor={0}
  >
    <View
      accessible={true}
      style={
        Object {
          "alignItems": "center",
          "borderColor": "#ABA2B0",
          "borderRadius": 15,
          "borderWidth": 1,
          "flexDirection": "row",
          "height": 50,
          "marginHorizontal": "5%",
          "marginVertical": 5,
          "opacity": 1,
          "padding": 10,
        }
      }
    >
      <Image
        height={20}
        source={
          Object {
            "testUri": "../../../src/asserts/images/email.png",
          }
        }
        style={
          Object {
            "marginLeft": 5,
          }
        }
        width={20}
      />
      <Text
        style={
          Object {
            "flex": 1,
            "fontFamily": "Gilroy SemiBold",
            "fontSize": 15,
            "paddingLeft": 5,
            "textAlign": "center",
          }
        }
      >
        Continue with Email
      </Text>
    </View>
  </RNGestureHandlerButton>
</View>
`;
